### PR TITLE
Add webpage layout

### PR DESCRIPTION
**Pull request by limavfabio**
This PR implements a copy of the template for the 2015 CC Global Summit webpage made by Cindy Shin.

Presentation video: https://www.loom.com/share/ebfb9e0b570d419ab183475b9f4446d4

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About - Indie Gaming</title>
+    <link rel="stylesheet" href="css/main.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```
Comment (keroloussamy) [inline about.html:15]: - [x] Kindly there's a horizontal scrolling on your about page, kindly fix it to match the original design.

image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+      >
+        <img
+          src="images/CreativeCommons_logo_trademark.svg"
+          alt="CC trademark logo"
+        />
+        <p class="ps-3 mb-0">
+          2022 Creative Commons <br />
+          Some Rights Reserved.
+        </p>
+      </footer>
```
Comment (keroloussamy) [inline about.html]: - [x] Kindly make the image smaller so you can give the text more space, to make your design much better.

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+              <a
+                class="nav-link px-md-3 px-2 active"
+                aria-current="page"
+                href="#"
+                >Home</a
+              >
+              <a class="nav-link px-md-3 px-2" href="about.html">About</a>
+              <a class="nav-link px-md-3 px-2" href="#">Program</a>
+              <a class="nav-link px-md-3 px-2" href="#">Join</a>
+              <a class="nav-link px-md-3 px-2" href="#">Sponsor</a>
```
Comment (keroloussamy) [inline index.html]: - [x] Kindly add the top short bar section with social media links, logout, and language links aligned to the right. To match the original layout.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @limavfabio, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [x] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by limavfabio** [comment]
@keroloussamy It's done bro, can I just submit it, or do I have to wait for you to undo the approval?

**Comment by keroloussamy** [comment]
Hello @limavfabio, Yes new everything is okay you can go forward.

**Pull request merged**